test/system/030-run.bats

```diff
@@ -513,6 +513,11 @@ json-file | f
     assert "$output" == "../usr/share/zoneinfo/Europe/Berlin" "localtime is linked correctly"
 }
 
+@test "podman run --tz with zoneinfo and custom TZDIR env" {
+    # Setting the timezone should fail because it does not exist
+    TZDIR="$PODMAN_TMPDIR/zoneinfo" run_podman 127 run --rm --tz Europe/Berlin "$SYSTEMD_IMAGE"
```

Comment:
Also, please don't just do error-status checks. Error messages must be confirmed:
```bash
    assert "$output" =~ "Error: setting timezone for container .*: stat $PODMAN_TMPDIR/zoneinfo/Europe/Berlin: no such file or directory"
```

Also, 
```bash
    skip_if_remote "TZDIR not propagated remotely"
```